src/SFA.DAS.ApplyService.Web/Views/Roatp/ManagementHierarchy/AddEditManagementHierarchy.cshtml

```diff
@@ -23,12 +26,12 @@
     <label for="FullName" class="govuk-label">Full name</label>
     @if (showErrors)
     {
-        <span class="govuk-error-message">
             @foreach(var nameErrorMessage in errorMessages.Where(x => x.Field == "FullName"))
             {
-                @Html.Raw(nameErrorMessage.ErrorMessage) <br />
+                <span class="govuk-error-message">
+                    @Html.Raw(nameErrorMessage.ErrorMessage)
```

Comment:
Not sure why this is being output as @Html.Raw, can you remove that bit?

Reply:
I've removed all the @Html.Raw()'s throughout these views.